mitodl/ocw-studio
# Replace "course_numbers" with "primary_course_number" for ocw import

#### Pre-Flight checklist


#### What are the relevant tickets?
N/A

#### What's this PR do?
Fixes a bug caused by "course_numbers" in OCW JSON being replaced with "primary_course_number" and "extra_course_numbers"

#### How should this be manually tested?
Tests should pass.
Run `docker-compose run web python manage.py import_ocw_course_sites --bucket ocw-to-hugo-output-qa  --filter 15-005`, it should complete and import several courses without errors.
